batch_convert.qml

```diff
@@ -593,27 +638,37 @@ MuseScore {
       var curScoreInfo = excerptsList.shift()
       var thisScore = curScoreInfo[0].partScore
       var partTitle = curScoreInfo[0].title
-      var fileBase = curScoreInfo[1]
-      var srcModifiedTime = curScoreInfo[2]
-
-      // create file base for part
-      var targetBase = fileBase + "-" + createDefaultFileName(partTitle)
+      var filePath = curScoreInfo[1]
+      var fileName = curScoreInfo[2]
+      var srcModifiedTime = curScoreInfo[3]
+
+      // create full file path for part
+      var targetBase;
+      if (differentExportPath.checked) {
```

Comment:
braces...

Reply:
I f I remove the braces here, do I put the statement on on single line? :
```
targetBase = targetFolderDialog.folderPath + "/" + fileName + "-" + createDefaultFileName(partTitle) + "." 
```
instead of 

```
targetBase = targetFolderDialog.folderPath + "/" + fileName 
                                    + "-" + createDefaultFileName(partTitle) + "." 
```

Reply:
Multiple lines with some indent is ok